homeassistant/components/vesync/const.py

```diff
@@ -0,0 +1,12 @@
+"""Constants for VeSync Component."""
+
+DOMAIN = 'vesync'
+VS_DISPATCHERS = 'vesync_dispatchers'
+VS_DISCOVERY = 'vesync_discovery_{}'
+SERVICE_UPDATE_DEVS = 'update_devices'
+
+CONF_PLATFORM = 'platform'
+CONF_LIGHTS = 'lights'
+CONF_SWITCHES = 'switches'
```

Comment:
Since this isn't a config option we shouldn't call it `CONF_`. Please rename the constant.